src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -290,9 +290,17 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
     /**
      * Adds the placeholder option to the list of select options if the list is empty.
      * @param selectOptions list containing all the options
+     * @param courses list of courses that the instructor have
      */
-    private void addPlaceholder(List<ElementTag> selectOptions) {
-        ElementTag placeholder = createOption("No active courses!", "", true);
+    private void addPlaceholder(List<ElementTag> selectOptions, List<CourseAttributes> courses) {
```

Comment:
It's not very readable why <code>addPlaceholder</code> would need a list of courses. 

I think it will be better if the logic for deciding which message to show is done at a level higher than this method